### PR TITLE
Refactors InputLookupSinopia and InputLookupQA

### LINKED ISSUE #1153: allow profile to limit inputLookupSInopia to Works or to Instances
ORIGINAL TICKET: Initially we want to support selected typeaheads that add three types of searches, BIBFRAME Works, Instances, and Items. Once [30](ttps://github.com/LD4P/sinopia_indexing_pipeline/issues/30) is fixed, we can support this functionality a couple of different ways:
-  Create specific Sinopia URNs for each of three classes (ie.e BIBFRAME Work urn:ld4p:sinopia:work)
- Use the same `urn:ld4p:sinopia` urn but add a subauthority value for the specific RDF class.

Pros and cons to both approaches.

Blocked by https://github.com/LD4P/sinopia_indexing_pipeline/issues/30

CHALLENGE:
Not all Works will be bf:Work, see notes below. It's also possible that some resources will have type that's a subclass of bf:Work. Would need to take those two possibilities into account.

### PR DESCRIPTION
**Pull request by justinlittman**
Refactors InputLookupSinopia and InputLookupQA to use common component. Adds limiting InputLookupSinopia searches by type.

closes #1153

### CODE REVIEW

**Comment by justinlittman** [comment]
image

File: src/components/editor/property/InputLookupQA.jsx [hunk 1]
```diff
@@ -85,107 +32,14 @@ const InputLookupQA = (props) => {
...
-                      onSearch={search}
-                      renderToken={(option, props, idx) => renderTokenFunc(option, props, idx)}
-                      {...typeaheadProps}
-                      filterBy={() => true}
-      />
-      {error && <span className="text-danger">{error}</span>}
-    </div>
-  )
-}
+    <React.Fragment>
```
Comment (jcoyne) [inline src/components/editor/property/InputLookupQA.jsx]: why do we need the `Fragment` here?

File: src/components/editor/property/InputLookupSinopia.jsx [hunk 2]
```diff
@@ -1,123 +1,47 @@
...
-                      placeholder={props.propertyTemplate.propertyLabel}
-                      minLength={1}
-                      filterBy={() => true }
-                      allowNew={() => true }
-                      id="sinopia-lookup" />
-
-      <span className="text-muted">Use a * to wildcard your search.</span>
-      {error && <span className="text-danger">{error}</span>}
-    </div>
+    <React.Fragment>
```
Comment (jcoyne) [inline src/components/editor/property/InputLookupSinopia.jsx]: Do we need Fragment here?

File: src/components/editor/property/InputLookupQA.jsx (hunk 1 above)
Comment (justinlittman) [reply src/components/editor/property/InputLookupQA.jsx]: No. Removed.

File: src/components/editor/property/InputLookupSinopia.jsx (hunk 2 above)
Comment (justinlittman) [reply src/components/editor/property/InputLookupSinopia.jsx]: No. Removed.

**Pull request merged**